lighthouse-core/audits/is-on-https.js

```diff
@@ -17,7 +17,9 @@ const UIStrings = {
   failureTitle: 'Does not use HTTPS',
   /** Description of a Lighthouse audit that tells the user *why* HTTPS use is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'All sites should be protected with HTTPS, even ones that don\'t handle ' +
-      'sensitive data. HTTPS prevents intruders from tampering with or passively listening ' +
+      'sensitive data. Sites should avoid having Mixed Content, where some resources are ' +
+      'loaded over HTTP despite the initial request being over HTTPS. HTTPS prevents ' +
+      'intruders from tampering with or passively listening ' +
```

Comment:
```suggestion
      'sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), ' +
      'where some resources are loaded over HTTP despite the initial request being served over HTTPS. ' +
      'HTTPS prevents intruders from tampering with or passively listening ' +
```

WDYT? might need to make some line length adjustments its hard to tell in GH

if you're not a fan of the wording changes, the "being *served* over" and the link would still be great :)

Reply:
I like it, especially the link.